models.tsx

```diff
@@ -0,0 +1,5 @@
+export interface Tournament {
```

Comment:
dope, yeah,  I should be adding expected API response formats here, once those are more nailed down I guess. I think `date` will be a relevant field here eventually